### PR TITLE
add ability to use a default function for modules

### PR DESCRIPTION
**Pull request by EnzoZafra**
# What Changed

Add a default function to use for modules. I made an assumption that the default to be used is the function that was in the README. This is:

```js
const defaultLocalizeFunction = (
  name: string,
  filePath: string,
  css: string
) => {
  const hash = crypto
    .createHash('md5')
    .update(css)
    .digest('hex')
    .slice(0, 6);
  return `${filePath || 'default'}-${name}-${hash}`;
}
```

I updated the README to reflect how to use this default function and replaced the 'custom' function in the README to avoid confusion.

# Why

https://github.com/intuit/postcss-themed/issues/25

Todo:

- [x] Add tests
- [x] Add docs
- [x] Add yourself to contributors (run `yarn contributors:add`)

### CODE REVIEW

**Comment by EnzoZafra** [comment]
Seems to be some token issue for the last stage of CircleCI. Let me know if I need to do anything about it!

File: README.md [hunk 1]
```diff
@@ -218,6 +218,23 @@ postcss([
...
+) => {
+  const hash = crypto
+    .createHash('md5')
+    .update(css)
+    .digest('hex')
+    .slice(0, 6);
+  return `${filePath || 'default'}-${name}-${hash}`;
+};
+```
+
```
Comment (hipstersmoothie) [inline README.md:237]: Can you add docs on what the configuration would look like?

**Review by hipstersmoothie — CHANGES_REQUESTED** [review]
This looks pretty good! just some docs updates and I can merge this

File: README.md (hunk 1 above)
Comment (EnzoZafra) [reply README.md:250]: sure thing!

**Review by hipstersmoothie — APPROVED** [review]

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in v2.1.0 :rocket:

**Pull request merged**